.github/workflows/build.yml

```diff
@@ -63,7 +63,9 @@ jobs:
           cache: 'gradle'
 
       - name: Build with Gradle
-        run: ./gradlew build -x test -PjdkVersion=8
+        run: |
+          ./gradlew build -x test
+          ./gradlew release -x test
```

Comment:
Could we use a seperate step like 
```
- name: Build release jar
  run: | 
     ./gradlew release
```

Reply:
There is something different, `build` build all packages with JDK17, while `release` builds client modules for JDK8